scripts/maintenance/audit_branches.py

```
@@ -0,0 +1,101 @@
+]
+
+# Essential security helpers that should not be reverted
+SECURITY_HELPERS = [
+    "hasDangerousProtocol",
+    "isRelativeUrl"
+]
+
+def run_command(args):
+    """Executes a git command securely using list-based arguments and shell=False."""
+    try:
+        result = subprocess.run(
```

Ruff (0.15.10)

[error] `subprocess` call: check for execution of untrusted input

(S603)